File: python/Day15/linked_list.py
def insert(self,head,data): 
  #Complete this method
  node = Node(data)

  #First element
  if head is None:
    head = node
    return head

  #All others
  temp = head
  while temp.next:
    temp = temp.next
  
  temp.next  = node
  return head

File: cpp/Day15/linked_list.cpp
Node* insert(Node *head,int data)
{
  //Complete this method
  Node* node = new Node(data);

  Node* temp = head;
  while(temp && (temp->next != NULL)){
    temp = temp->next;
  }

  if(head == NULL){
    head = node;
  }else{
    temp->next = node;   
  }

  return head;
}
